Src/PerformanceCollector/Filtering/AccumulatedValue.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Filtering
+{
+    using System.Collections.Concurrent;
+
+    /// <summary>
+    /// Accumulator for operationalized metrics.
```

Comment:
aggregated metrics? Preaggregated? Calculated? Segmented? Filtered? Again, operationalized for me is when you use it for alerting or dashboarding. Make it useful for "operations".